packages/core/src/components/popover/popover.tsx

```diff
@@ -271,16 +281,21 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
         }, className);
         targetProps.ref = this.refHandlers.target;
 
+        const childrenBaseProps = this.props.openOnTargetFocus && this.isHoverInteractionKind()
+            ? { tabIndex: 0 }
```

Comment:
this makes me wonder if this feature should wait until we can solve the popover target woes #724? should simplify this logic considerably.